Makefile

```diff
@@ -119,11 +119,17 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
+	
 .PHONY: lt-deploy
 lt-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/load_test | kubectl apply -f -
 
+.PHONY: local-deploy
+local-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
this make target is named a bit unclear to me as you are basically just deploying it with watcher, so maybe Call it `deploy-with-watcher`?